tests/ui/deprecated.stderr

```diff
@@ -30,11 +30,25 @@ error: lint `misaligned_transmute` has been removed: `this lint has been split i
 LL | #[warn(misaligned_transmute)]
    |        ^^^^^^^^^^^^^^^^^^^^
 
+error: unknown lint: `unused_collect`
+  --> $DIR/deprecated.rs:6:8
+   |
+LL | #[warn(unused_collect)]
+   |        ^^^^^^^^^^^^^^
+   |
+   = note: `-D unknown-lints` implied by `-D warnings`
+
 error: lint `str_to_string` has been removed: `using `str::to_string` is common even today and specialization will likely happen soon`
   --> $DIR/deprecated.rs:1:8
    |
 LL | #[warn(str_to_string)]
    |        ^^^^^^^^^^^^^
 
-error: aborting due to 6 previous errors
+error: unknown lint: `unused_collect`
```

Comment:
Is it expected that this triggers twice?

Reply:
Oh wait.. this is the wrong message. It should be the deprecation message. I'll take another look.